app/magtape/magtape.py

```diff
@@ -702,12 +748,16 @@ def send_slack_alert(
             timeout=5,
         )
 
-        app.logger.info(f"Slack Alert was successful ({slack_response.status_code})")
-        app.logger.debug(f"Slack API Response: {slack_response}")
+        app.logger.info(
+            f"Slack Alert ({slack_target_type}) was successful ({slack_response.status_code})"
+        )
+        app.logger.debug(f"Slack API Response ({slack_target_type}): {slack_response}")
```

Comment:
unclear to me if this code may execute when the `request.post` throws an exception. Maybe clearer to have an `else:` block at the end of your `try:` block like this:
```python
# X is not defined so the try block throws an exception on the print
try:
  print(x)
except:
  print("Exception, x is not defined")
else:
  print("Nothing went wrong")
  
print("=====================")

x = "OK"

# The try block does not raise any errors now because x is defined, so the else block is executed:
try:
  print(x)
except:
  print("Exception, x is not defined")
else:
  print("Nothing went wrong")
```

Reply:
Fixed this